apps/meteor/client/components/message/Attachments/QuoteAttachment.tsx

```diff
@@ -25,42 +32,57 @@ const hover = css`
 	}
 `;
 
-export const QuoteAttachment: FC<MessageQuoteAttachment> = ({
-	author_icon: url,
-	author_name: name,
-	author_link: authorLink,
-	message_link: messageLink,
-	ts,
-	text,
-	attachments,
-}) => {
+type QuoteAttachmentProps = {
+	attachment: MessageQuoteAttachment;
+	message: IMessage;
+};
+
+export const QuoteAttachment = ({ attachment, message }: QuoteAttachmentProps): ReactElement => {
 	const format = useTimeAgo();
+
+	const msg: IMessage = {
+		_id: message._id,
+		ts: message.ts,
+		u: message.u,
+		_updatedAt: message._updatedAt,
+		rid: message.rid,
+		md: undefined,
+		msg: attachment.text,
+	};
+
+	const tokens = useParsedMessage(msg);
+
 	return (
 		<>
-			<AttachmentContent className={hover} width='full'>
+			<AttachmentContent className={quoteStyles} width='full'>
 				<AttachmentDetails
 					is='blockquote'
 					borderRadius='x2'
 					borderWidth='x2'
 					borderStyle='solid'
 					borderColor='neutral-200'
-					borderInlineStartColor='neutral-600'
+					borderInlineStartColor='neutral-500'
 				>
 					<AttachmentAuthor>
-						<AttachmentAuthorAvatar url={url} />
-						<AttachmentAuthorName {...(authorLink && { is: 'a', href: authorLink, target: '_blank', color: undefined })}>
-							{name}
+						<AttachmentAuthorAvatar url={attachment.author_icon} />
+						<AttachmentAuthorName
+							{...(attachment.author_name && { is: 'a', href: attachment.author_link, target: '_blank', color: 'info' })}
+						>
+							{attachment.author_name}
 						</AttachmentAuthorName>
-						{ts && (
-							<Box fontScale='c1' {...(messageLink ? { is: 'a', href: messageLink } : { color: 'hint' })}>
-								{format(ts)}
+						{attachment.ts && (
+							<Box
+								fontScale='c1'
+								{...(attachment.message_link ? { is: 'a', href: attachment.message_link, color: 'hint' } : { color: 'hint' })}
+							>
+								{format(attachment.ts)}
 							</Box>
 						)}
 					</AttachmentAuthor>
-					<MarkdownText parseEmoji variant='document' content={text} />
-					{attachments && (
+					{tokens ? <MessageMarkup tokens={tokens} /> : <MarkdownText parseEmoji variant='document' content={attachment.text} />}
```

Comment:
Why keep MarkdownText? Since our markup has similar tokens, won't `tokens` always be true if there's any text in the quote?